qutebrowser/utils/usertypes.py

```diff
@@ -113,18 +114,39 @@ def _snap_in(self, offset):
             True if the value snapped in (changed),
             False when the value already was in the list.
         """
-        op = operator.le if offset < 0 else operator.ge
+        optype = operator.le if offset < 0 else operator.ge
+        currentval = None
+        items = None
+
+        is_int = lambda s: isinstance(self.items[self._idx], int) \
+                or s.lstrip("-+").isdigit()
```

Comment:
I don't understand why you added this check - I see some `test_cmdhistory.py` tests are failing if I remove it, but I don't get why (though it's been a while since I last looked at `NeighborList` code).